lib/services/argument-handler.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = argumentHandlerServiceFactory;
```

Comment:
in the index.js of on-core, so it can take effect for all app

Felix

-----Original Message-----
From: jlongever [notifications@github.com]
Received: 星期五, 29 1月 2016, 21:57
To: RackHD/on-core [on-core@noreply.github.com]
CC: Yuan, Felix [Felix.Yuan@emc.com]
Subject: Re: [on-core] improve log output (color/level/caller) (#66)


In lib/services/argument-handler.js<https://github.com/RackHD/on-core/pull/66#discussion_r51262428>:

> @@ -0,0 +1,60 @@
> +// Copyright 2016, EMC, Inc.
> +
> +'use strict';
> +
> +module.exports = argumentHandlerServiceFactory;


@yyscamper<https://github.com/yyscamper> This looks like a new service. Where does the ArgumentHandlerService get used?

—
Reply to this email directly or view it on GitHub<https://github.com/RackHD/on-core/pull/66/files#r51262428>.